src/test/java/org/jenkinsci/plugins/badge/actions/EmbeddableBadgeConfigsActionTest.java

```diff
@@ -0,0 +1,53 @@
+package org.jenkinsci.plugins.badge.actions;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsNull.nullValue;
+
+import hudson.model.Job;
+import hudson.model.Run;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
```

Comment:
Don't need this object since no mocking is required for the valid tests in the file.
```suggestion
```